barretenberg/cpp/src/barretenberg/honk/composer/ultra_composer.test.cpp

```diff
@@ -884,6 +868,66 @@ TEST_F(UltraHonkComposerTests, ram)
     prove_and_verify(circuit_builder, composer, /*expected_result=*/true);
 }
 
+TEST_F(UltraHonkComposerTests, range_checks_on_duplicates)
```

Comment:
Yeah there's no meaningful change, I'm just brining this outdated test up to the modern standard. The logic of what we now call a circuit builder used to be contained in the "composers". The logic is otherwise the same.